src/Entity/User.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * @ORM\Table(name="app_user")
+ * @ORM\Entity(repositoryClass="App\Repository\User\UserRepository")
+ */
+class User implements UserInterface
+{
+    /**
+     * @ORM\Id()
+     * @ORM\Column(type="guid", unique=true)
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=100, unique=true)
```

Comment:
Are you sure emails are limited to 100 characters?
[the maximum length limit of an email address is 254 not 320](https://7php.com/the-maximum-length-limit-of-an-email-address-is-254-not-320/)
I'd still use a 320.